lab-catherine/__test__/arithmetic.test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const arithmetic = require('../lib/arithmetic');
+
+
+describe('arithmetic.test.js', () => {
+  describe('arithmetic.sum', () => {
+    test('does arithmetic.sum work properly', () => {
+      expect(arithmetic.sum('nine', 'one')).toBe(null);
```

Comment:
it would also be a good idea for both add and sub to test for one argument as a number input and the other as non numeric input, one of each.